Packs/Armis/Integrations/ArmisEventCollector/ArmisEventCollector.yml

```diff
@@ -54,6 +54,14 @@ configuration:
   defaultvalue: "1"
   type: 19
   required: false
+- section: Collect
+  advanced: true
+  display: Minutes to delay
+  additionalinfo: Number of minutes to delay when fetching events (to handle events creation delay), default is 0.
+  name: fetch_delay
+  defaultvalue: "0"
+  type: 0
```

Comment:
thinking now maybe the default shouldnt be 0, @ShahafBenYakir wdyt should we have the delay be default behavior?